src/rust/ensogl/src/display/shape/text/text_field/cursor.rs

```diff
@@ -470,9 +481,9 @@ impl Cursors {
     ///
     /// If after this operation some of the cursors occupies the same position, or their selected
     /// area overlap, they are irreversibly merged.
-    pub fn navigate_cursors<Predicate>
-    (&mut self, navigation:&mut CursorNavigation, step:Step, mut predicate:Predicate)
-    where Predicate : FnMut(&Cursor) -> bool {
+    pub fn navigate_cursors<P>
+    (&mut self, navigation:&mut CursorNavigation, step:Step, mut predicate:P)
+    where P : FnMut(&Cursor) -> bool {
         let filtered = self.cursors.iter_mut().filter(|c| predicate(c));
```

Comment:
Would it be possible to change it to `.filter(predicate)` instead?

Reply:
:+1: